notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            return
</pre><p>You can't return <code>None</code> as the caller expects a pd.Dataframe.</p><p>I suggest removing the <code>try catch</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Modifying to just `raise` if an error occurs in try catch:

```
    try:
        job_config = bigquery.QueryJobConfig(dry_run=True, use_query_cache=False)

        bq_client.query(sql, job_config=job_config)

    except BadRequest as err:
        raise
```

So this returns an error if, for example, the SQL query uses improper syntax (or any other error from BigQuery).

Reply:
<p>If you just raise the exception, isn't that the same behaviour as removing the try-catch?</p>